### ISSUE TITLE
Using Middleware; redirect instead of throw exception

### ISSUE THREAD

**Issue by Naoray** [santigarcor/laratrust#21]
Wouldn't it be nicer if we would use a redirect to a specifiable route instead of just calling `abort(403)`. Or maybe make it optional?!

We could ad a config entry like
```php
'middleware_handling' => 'abort',
'middleware_params' => '403',
```
and change `abort(403)` in Middleware to
```php
call_user_func(config('laratrust.midleware_handling'), config('middleware_params));
```

=> just an idea!

**Comment by santigarcor** [comment]
Actually i think it's a good idea

**Comment by Naoray** [comment]
I tried it with:
```php
// config
'middleware_handling' => 'abort',
'middleware_params' => '403',


// middleware
if ($this->auth->guest() || !$request->user()->ability($roles, $permissions, [ 'validate_all' => $validateAll ]))
{
    return call_user_func(Config::get('laratrust.middleware_handling'), Config::get('laratrust.middleware_params'));
}
```
... it works, but there are some errors in the tests:
```
1) LaratrustAbilityTest::testHandle_IsGuestWithNoAbility_ShouldAbort403
Mockery\Exception\NoMatchingExpectationException: No matching handler found for Mockery_9__config::get('laratrust.middleware_handling'). Either the method was unexpected or its arguments matched no expected argument list for this method
```

But I did not figure out how to resolve it yet. Any idea?

**Comment by santigarcor** [comment]
@Naoray Indeed i'm working on it. I think the problem is the test.

**Comment by santigarcor** [comment]
Could you please try it out, i got it working in f34f223eb3af5b4dada745c28948cabe61b82adb. remember to add the `middleware_handling` and `middleware_params` to your `config/laratrus.php` file

**Comment by Naoray** [comment]
you've had a type in the middlewares... fixed it in #24 . But the tests are still failing!! :/

**Comment by santigarcor** [comment]
@Naoray yeah i had the typo, maybe i did the typo in the tests too, i'm going to check it out

**Comment by santigarcor** [comment]
@Naoray Yeah the problem was that the typo was on the tests too. i'm solving it now

**Comment by Naoray** [comment]
@santigarcor got them! fixed in #25

**Comment by santigarcor** [comment]
@Naoray , Thanks but i already solved it with #934f260c063dc150274c00336e6a1d3b23a40ff5
